jsonschema2pojo-integration-tests/src/test/java/org/jsonschema2pojo/integration/config/IncludeJsr303AnnotationsIT.java

```diff
@@ -182,6 +182,23 @@ public void jsr303SizeValidationIsAddedForSchemaRuleMinItemsAndMaxItems() throws
 
     }
 
+    @Test
+    public void jsr303EmailValidationIsAddedForFormatEmailSchemaRule() throws ClassNotFoundException {
```

Comment:
A non-related "issue": would it make sense to fix typo by renaming `jar303AnnotationsValidatedForAdditionalProperties` to `jsr303AnnotationsValidatedForAdditionalProperties` ?